src/components/tabs/tabs.md.scss

```diff
@@ -54,14 +54,14 @@ $tabs-md-tab-text-transition:                 transform .3s ease-in-out !default
 /// @prop - Transform for the active tab button icon when the layout is icon-top, icon-only, or title-only
 $tabs-md-tab-icon-transform-active:           translate3d(0, -2px, 0) !default;
 
-/// @prop - Transform for the active tab button icon when the layout is icon-right
-$tabs-md-tab-icon-right-transform-active:     translate3d(2px, 0, 0) !default;
+/// @prop - Transform for the active tab button icon when the layout is icon-end
+$tabs-md-tab-icon-end-transform-active:     translate3d(2px, 0, 0) !default;
 
 /// @prop - Transform for the active tab button icon when the layout is icon-bottom
 $tabs-md-tab-icon-bottom-transform-active:    translate3d(0, 2px, 0) !default;
 
-/// @prop - Transform for the active tab button icon when the layout is icon-left
-$tabs-md-tab-icon-left-transform-active:      translate3d(-2px, 0, 0) !default;
+/// @prop - Transform for the active tab button icon when the layout is icon-start
+$tabs-md-tab-icon-start-transform-active:      translate3d(-2px, 0, 0) !default;
```

Comment:
@AmitMY Sorry he was just using that as an example variable. So basically if we rename:

`$tabs-md-tab-icon-left-transform-active` to `$tabs-md-tab-icon-start-transform-active` 

(or any other variables) it will be considered a breaking change if any users are overriding these variables. It's an easy fix, sure, but it's still technically breaking and would prevent us from merging this until the 4.0 release. We're suggesting doing the following:

```
// deprecated
$tabs-md-tab-icon-left-transform-active:            translate3d(-2px, 0, 0) !default;

/// @prop - 
$tabs-md-tab-icon-start-transform-active:           $tabs-md-tab-icon-left-transform-active !default;

in-the-selector-the-variable-is-used {
  transform: $tabs-md-tab-icon-start-transform-active;
}
```

This would introduce `*-start` as the new variable (and it would be documented) but if they are using the `*-left` variable to override styles it would still be used. Does that make sense? 🙂